Dockerfile

```diff
@@ -35,31 +22,33 @@ RUN APT_INSTALL="apt-get install -y --no-install-recommends" && \
         wget \
         libncurses5-dev \
         libncursesw5-dev \
+        gcc \
+        make \
         cmake \
         nano \
         tmux \
         htop \
         ssh \
-        # OpenCV
-        libsm6 libxext6 libxrender-dev \
         && \
+        # NVTop >>
+        git clone --depth 1 --branch 1.2.2 -q https://github.com/Syllo/nvtop.git nvtop && \
```

Comment:
Why `--depth 1` as opposed to `--depth 0`?

Reply:
good question, I've just tested it:
![image](https://user-images.githubusercontent.com/32543098/129542197-350a6cfc-f6e1-4f2f-a3fc-3fa6239dfd55.png)
its just the number of commits to fetch

Reply:
Wow, I was sure `--depth 0` works. Thanks for testing!